spec/numberformat.html

```diff
@@ -30,9 +30,12 @@ <h1>SetNumberFormatDigitOptions ( _intlObj_, _options_, _mnfdDefault_, _mxfdDefa
           1. Set _intlObj_.[[MaximumSignificantDigits]] to _mxsd_.
         1. Else if _mnfd_ is not *undefined* or _mxfd_ is not *undefined*, then
           1. Set _intlObj_.[[RoundingType]] to ~fractionDigits~.
-          1. Let _mnfd_ be ? DefaultNumberOption(_mnfd_, 0, 20, _mnfdDefault_).
-          1. Let _mxfdActualDefault_ be max( _mnfd_, _mxfdDefault_ ).
-          1. Let _mxfd_ be ? DefaultNumberOption(_mxfd_, _mnfd_, 20, _mxfdActualDefault_).
+          1. Let _specifiedMnfd_ be ? DefaultNumberOption(_mnfd_, 0, 20, *undefined*).
+          1. Let _specifiedMxfd_ be ? DefaultNumberOption(_mxfd_, 0, 20, *undefined*).
+          1. If _specifiedMxfd_ is not *undefined*, set _mnfdDefault_ to min(_mnfdDefault_, _specifiedMxfd_).
+          1. Set _mnfd_ to ! DefaultNumberOption(_specifiedMnfd_, 0, 20, _mnfdDefault_).
```

Comment:
I do not understand how would this work:
If _mnfd_ is undefined, then before the change DefaultNumberOption would set _mnfd_ to  _mnfdDefault_ but with this change DefaultNumberOption would now set _mnfd_ to undefined. Then if _mxfd_ is not undefined, DefaultNumberOption would set _specifiedMxfd_ to an value which is not undefined.  and therefore the if condition will be true and execute set _mnfdDefault_ to min(_mnfdDefault_, _specifiedMxfd_). 
but now we know _specifiedMxfd_ could be undefined as above, so how to run the min between _mnfdDefault_ and undefined?